src/main/scala/rocket/RocketCore.scala

```diff
@@ -50,7 +50,8 @@ case class RocketCoreParams(
   fpu: Option[FPUParams] = Some(FPUParams()),
   debugROB: Boolean = false, // if enabled, uses a C++ debug ROB to generate trace-with-wdata
   haveCease: Boolean = true, // non-standard CEASE instruction
-  haveSimTimeout: Boolean = true // add plusarg for simulation timeout
+  haveSimTimeout: Boolean = true, // add plusarg for simulation timeout
+  traceWdata: Boolean = false
 ) extends CoreParams {
```

Comment:
Instead of `traceWdata flag`, this should have `debugROB: Option[DebugROBParams]` 
`case class DebugROBParams(size: Int)`

If size < 1, use the SW ROB.